content/influxdb/cloud-dedicated/reference/client-libraries/v2/go.md

```diff
@@ -16,8 +16,7 @@ Use the [InfluxDB Go client library](https://github.com/influxdata/influxdb-clie
 {{% note %}}
```

Comment:
Wondering if this should be a warning instead of a note. Not super opinionated about it though 😄